src/ur5e_arm.hpp

```diff
@@ -28,20 +29,33 @@ const std::string OUTPUT_RECIPE = "/src/control/rtde_output_recipe.txt";
 const std::string INPUT_RECIPE = "/src/control/rtde_input_recipe.txt";
 
 // locations of log files that will be written
-const std::string TRAJECTORY_LOG = "/logs/trajectory.csv";
-const std::string WAYPOINTS_LOG = "/logs/waypoints.csv";
+const std::string TRAJECTORY_CSV_NAME_TEMPLATE = "/%1%_trajectory.csv";
+const std::string WAYPOINTS_CSV_NAME_TEMPLATE = "/%1%_waypoints.csv";
+const std::string ARM_JOINT_POSITIONS_CSV_NAME_TEMPLATE = "/%1%_arm_joint_positions.csv";
 
 // TODO: using this is deprecated by the URCL, we could find some way around using it
 const std::string CALIBRATION_CHECKSUM = "calib_12788084448423163542";
 
 // constants for robot operation
-const int NOOP_DELAY = 100000;  // 100 milliseconds
-const double TIMESTEP = 0.2;    // seconds
+const float TIMESTEP = 0.2f;  // seconds
+const int NOOP_DELAY = 1000;  // 1 millisecond
```

Comment:
another thing we should check, the arm has a log that states

`Setting up RTDE communication with frequency 500.0` Hz

so we should double check that setting this to 1 ms doesn't cause any issues

Reply:
that makes sense. I reduced this as when you and olivia were testing you were reporting pipe errors. I decreased this delay to try to address that. Would you like me to revert back to the previous 100 ms delay?

Reply:
we spoke offline for this, fine to leave as is until we notice an issue/investigate if we want to change the client frequency as a whole